trip/views.py

```diff
@@ -1,3 +1,37 @@
-from django.shortcuts import render  # noqa
+from django.shortcuts import render, redirect  # noqa
+from django.contrib.auth.decorators import login_required
+from django.contrib import messages
+from django.urls import reverse
+
+from . import forms
+from .models import Trip
+
 
 # Create your views here.
+@login_required
+def create_trip(request):
+    if request.method == "POST":
+        ut_creation_form = forms.UserTripCreationForm(request.POST)
```

Comment:
Nit; Can we make ut as userTrip everywhere as it will be easier for everyone who reads the code post this